kubernetes/internal/create-weblogic-domain.sh

```diff
@@ -584,6 +605,30 @@ function createDomain {
 
 }
 
+#
+# Deploy voyager/HAProxy load balancer
+#
+function setupVoyagerLoadBalancer {
+  # deploy voyager ingress controller
+  kubectl create namespace voyager
+  curl -fsSL https://raw.githubusercontent.com/appscode/voyager/6.0.0/hack/deploy/voyager.sh \
+  | bash -s -- --provider=baremetal --namespace=voyager
+
+  # deploy voyager ingress resource
+  kubectl create -f ${voyagerOutput}
+
+  echo Checking voyager deploy
+  vdep=`kubectl get deploy | grep voyager | wc | awk ' { print $1; } '`
```

Comment:
This is a pretty weak check.  It only validates that the deployment exists, but not that its running properly.